### ISSUE TITLE
Replace login page redirect with home page redirect

### ISSUE THREAD

**Issue by nilbus** [nilbus/handoff#16]
Visiting an inner page when not logged in brings you to a useless login page, for which no one will know credentials.

**Comment by corey-f** [comment]
I see that an "authenticate_user" action is mentioned in "application_controller.rb" and "welcome_controller.rb", but I don't see an actual "authenticate_user" method anywhere. Any thoughts on where I can find the code that "brings you to a useless login page"?

I tried making a route as a work around, but that was not effective
get '/users/sign_in', to: redirect('welcome#index')

Also I'm curious how all of these additional routes are getting added into the app when they are not in the routes.rb file. Such as:

   new_user_registration_path > users/sign_in(.:format) > devise/sessions#new

There is no devise folder or controller in the controllers folder, so this appears to be quite the mystery to me.

Addressing this issue will require understanding what devise is and how it works. I didn't find the devise.rb file in initializers very helpful, other than to offer some file trace in the app that devise existed. According to git hub devise.rb was added by Edward.
